volcanic_island_model_demo.ipynb

```diff
@@ -47,6 +47,10 @@
     "        'run_duration': 1000.0,\n",
     "        'timestep_size': 1.0\n",
     "    },\n",
+    "    'output': {\n",
+    "        'output_interval_fraction': 0.05,  # e.g., 0.05 means 5% intervals: 5, 10, 15, 20...100%\n",
```

Comment:
so it sounds like the output will create a series of images (netcdf files), which could be stitched together to make an animation? If that's the idea, very cool!

Reply:
Yes, and they can be read into ParaView to make animations